modular_skyrat/modules/gladiator/code/game/objects/items/gladiator_items.dm

```diff
@@ -19,22 +27,85 @@
 	if(.)
 		incomingchance.block_chance -= bonus_value
 
-/obj/item/claymore/agateram //it just works
-	name = "ancient blade agateram"
-	desc = "A millenia-old blade made from a material that you can't even begin to fathom. It flows with the power of the Marked One who once held it." //That thing was too big to be called a sword. Too big, too thick, too heavy, and too rough, it was more like a large hunk of iron.
-	icon = 'modular_skyrat/master_files/icons/obj/agateram.dmi'
-	icon_state = "demonsword"
-	inhand_icon_state = "demonsword"
-	lefthand_file = 'modular_skyrat/master_files/icons/mob/agateraminhandsleft.dmi'
-	righthand_file = 'modular_skyrat/master_files/icons/mob/agateraminhandsright.dmi'
+/obj/item/clothing/neck/warrior_cape
+	name = "\proper cloak of the marked one"
+	desc = "A cloak worn by those that have faced death in the eyes and prevailed. <b><p style='color:red;'>Struggle against the tide, no matter how strong it may be.</p></b>"
```

Comment:
Why did you use raw HTML in there? Just use something like `span_warning()` instead.